### ISSUE TITLE
Consider adding Jupyter Notebook build artefacts and MyBinder integration

### ISSUE THREAD

**Issue by jni** [elegant-scipy/elegant-scipy#174]
It would be amazing for people to just be able to go to the repo and launch a MyBinder instance containing all the book notebooks!

However, I think we probably would need to update the exercise excising machinery so that the notebooks themselves also don't contain solutions.

**Comment by hdashnow** [comment]
Yes! At PyCon AU sprint I saw MyBinder used to run programming workshops for humanities students (who had a lot of trouble installing all the things on Windows). It looks very cool.
